interface-definitions/interfaces-wirelessmodem.xml.in

```diff
@@ -45,6 +45,7 @@
           #include <include/interface-description.xml.i>
           #include <include/interface-disable.xml.i>
           #include <include/interface-vrf.xml.i>
+          #include <include/interface-mirror.xml.i>
```

Comment:
Does this actually work? WWAN interfaces are kinda special.

Reply:
Where is it special? I can delete first